typescript/.eslintrc.json

```diff
@@ -1,5 +1,30 @@
 {
+  "parser": "@typescript-eslint/parser",
   "extends": [
-    "plugin:import/typescript"
+    "../base/.eslintrc.json",
+    "plugin:import/typescript",
+    "prettier/@typescript-eslint",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended"
+  ],
+  "plugins": ["@typescript-eslint"],
+  "rules": {
+    "@typescript-eslint/no-explicit-any": 1,
+    "@typescript-eslint/explicit-function-return-type": 0,
+    "@typescript-eslint/semi": 2,
+    "@typescript-eslint/interface-name-prefix": [
+      2,
+      {
+        "prefixWithI": "never"
+      }
+    ]
+  },
+  "overrides": [
+    {
+      "files": ["**/*.{ts,tsx}"],
+      "rules": {
+        "react/prop-types": "off"
```

Comment:
Are we not ditching prop types all together?

Reply:
This is overriding the rule for typescript files!

Reply:
Yes, but are we not ditching prop-types all together? They're just annoying, we shouldn't have any rules for them IMO.

Reply:
That's for sure, removing..